ericberringer/Currency-Kidz
# build: seeded database, all tables and data were created properly.

Description Fixes # (issue)

Type of change:

Basic Setup, seeded database with one item per fixture.

Testing Instructions:
Select * on my tables to check data was present

git fetch --all 
git checkout CK-Setup2

Checklist:

X (I have preformed a self review)

(Code follows style guidelines of project)

(I have commented my code. Especially w/ hard to understand code)

(Changes I have made generate no new warnings or errors)

(I have added test instructions that prove my fix is effective or that my feature works)

(I am not ruminating about the things my therapist told me to forget about b/c I cannot do anything about them. For example, the state of the world. )

- Fixes #
